api/ReportBody.json

```
@@ -0,0 +1,102 @@
+{
+  "api": {
+    "Report": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/ReportBody",
+        "spec_url": "https://w3c.github.io/reporting/#reportbody",
+        "support": {
+          "chrome": {
+            "version_added": "69"
+          },
+          "chrome_android": {
+            "version_added": "69"
+          },
+          "edge": {
+            "version_added": "79"
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "56"
+          },
+          "opera_android": {
+            "version_added": "48"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": "10.0"
+          },
+          "webview_android": {
+            "version_added": "69"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "toJSON": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Report/toJSON",
```

```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ReportBody/toJSON",
```

Except, I don't think this page will ever be created, so should `mdn_url` just be omitted here? I know @sideshowbarker feels pretty strongly that we shouldn't include broken URLs. That's tricky to guarantee in general, but in this case it's easy to predict and avoid.